### DISCUSSION TITLE
[Crash] Arc 3.0 and after Character crashes

### DISCUSSION THREAD

**Discussion by JarlTheKing** [Raytwo/ARCropolis#339] [Issues] [answered]
System Firmware Version
14.1.1

Atmosphere Version
1.3.2

Smash Version
13.0.1

ARCropolis Version
3.3.0

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
 During the loading for a battle
 
Details:
Ever since updating to Arc 3.0 (and after) whenever I choose certain characters with skins (or other mods on them) the game crashes on the loading screen to a match. So instead I have to update back to Arc 2.1.1 in order for everything to work properly again. I'm still new in working around mods, and I would like to add all the cool mods that only really work with the new Arc updates.

Crash report
[01651764735_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/8655930/01651764735_01006a800016e000.log)
![166971119-1f988b64-9189-452e-9945-22538e316024](https://user-images.githubusercontent.com/102709826/167513317-7c89658d-332a-455f-9060-f1871656f68e.jpg)

**Comment by Raytwo** [comment, marked as answer]
Apparently it's one of your effect files causing an issue, time to disable them until you find the problematic one!
